.gitignore

```diff
@@ -78,7 +76,4 @@ node/
 node_modules/
 
 # Generated JavaScript Bundles
-jsbundles
```

Comment:
Using a more specific path to `jsbundles`, since there is only a single such directory.